spec/spec.md

```diff
@@ -333,6 +333,17 @@ be ignored, unless otherwise specified by a [[ref:Feature]];
   The value of the `id` property ****MUST**** be a string that does not
   conflict with the `id` of another [[ref:Input Descriptor Object]] in the same
   [[ref:Presentation Definition]].
+- The [[ref:Input Descriptor Object]] ****MUST**** contain an `intent_to_retain`
+  property.
+  The value of the `intent_to_retain` property ****MUST**** be a boolean that 
+  indicates the [[ref:Verifier]] intents to retain the [[ref:Claim]]'s data being
+  requested.
+  A [[ref:Verifier]] shall not retain any data, including digests, signatures,
```

Comment:
Is "shall" here intentional? should it be "must not retain"?

Reply:
Changed